js/app/dashboard/components/LeafletMap.js

```diff
@@ -37,7 +44,44 @@ class LeafletMap extends Component {
         this.proxy.hidePolyline(username)
       }
     })
+
+    if (prevProps.showFinishedTasks !== this.props.showFinishedTasks) {
+      const finishedTasks = _.filter(tasks, task => task.status !== 'TODO')
+      if (this.props.showFinishedTasks) {
+        _.forEach(finishedTasks, task => this.proxy.addTask(task))
+      } else {
+        _.forEach(finishedTasks, task => this.proxy.hideTask(task))
+      }
+    }
+
+    if ( prevProps.selectedTags !== this.props.selectedTags ) {
+
+
+      let toAdd = _.map(_.filter(this.props.selectedTags, tag => prevProps.selectedTags.indexOf(tag) < 0), tag => tag.name)
```

Comment:
!! this variables are never used, as they are redefined in the for loop just after